scout/export/variant.py

```diff
@@ -2,17 +2,19 @@
 import logging
 import urllib.parse
 
+from scout.adapter.mongo.base import MongoAdapter
 from scout.constants import CHROMOSOME_INTEGERS
+from scout.models.managed_variant import ManagedVariant
 
 LOG = logging.getLogger(__name__)
 
 
-def export_variants(adapter, collaborator, document_id=None, case_id=None):
+def export_variants(
+    adapter: MongoAdapter, collaborator: str, document_id: str = None, case_id: str = None
+) -> dict:
     """Export causative variants for a collaborator
 
     Args:
-        adapter(MongoAdapter)
-        collaborator(str)
         document_id(str): Search for a specific variant
```

Comment:
You could also remove the Args thing and provide a better docstring for this function.